keywords.txt

```diff
@@ -9,19 +9,22 @@ LSM6DS3	KEYWORD1
 IMU	KEYWORD1
 
 #######################################
-# Methods and Functions 
-#######################################	
+# Methods and Functions
+#######################################
 
 begin	KEYWORD2
 end	KEYWORD2
 
 readAcceleration	KEYWORD2
 readGyroscope	KEYWORD2
+readTemperature KEYWORD2
 gyroscopeAvailable	KEYWORD2
 accelerationAvailable	KEYWORD2
+temperatureAvailable  KEYWORD2
 accelerationSampleRate	KEYWORD2
 gyroscopeSampleRate	KEYWORD2
+temperatureSampleRate KEYWORD2
```

Comment:
```suggestion
readTemperature	KEYWORD2
gyroscopeAvailable	KEYWORD2
accelerationAvailable	KEYWORD2
temperatureAvailable	KEYWORD2
accelerationSampleRate	KEYWORD2
gyroscopeSampleRate	KEYWORD2
temperatureSampleRate	KEYWORD2
```
The Arduino IDE only recognizes the keywords when you use a true tab separator here, not spaces.

Reply:
Sorry about that, I tried to use tab.